Dispatcher/Constants/APIConstants.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Constants.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 30/04/2023.
+//
+
+import Foundation
+
+struct APIConstants {
+    static let newscatcherURL = "https://api.newscatcherapi.com/v2/latest_headlines"
+    static let APIkey = "sOEtokE8TQCbCtBWifqf87OgISACtNadOaVf88T9vAY"
```

Comment:
It is not safe to store sensitive data such as API keys like this, especially as it is being uploaded to github and is available for everyone to abuse.
A better solution - create a JSON file (e.g. Config.json) to store the key inside. To use the key in your project just read the API key value from the file. Don't forget to add the json file into your .gitignore so it won't be uploaded to github.